tools/bridge/bridge/contract_validation.py

```diff
@@ -0,0 +1,59 @@
+from web3 import Web3
+from web3.contract import Contract
+from web3._utils.abi import abi_to_signature
+from eth_utils import to_checksum_address
+
+from bridge.contract_abis import MINIMAL_VALIDATOR_PROXY_ABI
+
+
+def validate_contract(contract: Contract) -> None:
```

Comment:
maybe `validate_contract_existence` as it doesn't really validate the contract itself, just that the right contract exists at the right address.

Reply:
I think you suggestions is good enough. I could argue that there is even more included, but I guess a matching ABI could be also count as the "correct" existence.

Reply:
:heavy_check_mark: